zh/backup-to-aws-s3-using-br.md

```diff
@@ -475,6 +477,7 @@ Stopped 是日志备份的终止状态，此状态下无法再次更改状态，
       namespace: backup-test
     spec:
       backupMode: log
+      logSubcommand: log-start/log-pause/log-stop
```

Comment:
What sub-command should the user choose here?

Reply:
any one

Reply:
can it truncate log even with `log-pause`/`log-stop` sub command?

Reply:
yes